Dockerfile

```diff
@@ -0,0 +1,36 @@
+## Build
+FROM golang:1.13-buster AS build
```

Comment:
I would suggest using at least go 1.15.

Now that go 1.17 has been released (earlier this week), 1.16 would actually be a safe/stable option.

Reply:
i tried go 1.16 and got some errors, probably would need to update some code.